esp/platform/espprotocol.cpp

```diff
@@ -755,3 +755,24 @@ int CEspProtocol::countBindings(int port)
     else
         return apport->getBindingCount();
 }
+
+bool checkEspConnection(IEspContext& ctx)
+{
```

Comment:
Should you check if ctx is ok before this cast ?

Reply:
No it's guaranteed to be not null where it's going to be called (reason it's passed in as reference both here and the service method)

Reply:
ok